hack/convert-output-to-tsv.py

```diff
@@ -71,94 +70,82 @@ def printd(x):
                 # hles_health_condition: read + copy dog_id and hs_condition and hs_condition_is_congenital
                 # concatenate and write out as pk_name
                 elif table_name == "hles_health_condition":
-                    entity_name = pk_prefix + table_name + '_id'
+                    entity_name = f"{PRIMARY_KEY_PREFIX}:{table_name}_id"
                     # grab the congenital flag and convert to int
                     congenital_flag = row.get('hs_condition_is_congenital')
-                    if congenital_flag == True:
-                        congenital_flag = 1
-                    elif congenital_flag == False:
-                        congenital_flag = 0
-                    else:
-                        print (f"Error, 'hs_condition_is_congenital' is not populated in {table_name}")
-                    row[entity_name] = ('%s-%s-%s' % 
-                        (row.get('dog_id'), row.get('hs_condition'), congenital_flag))
+                    try:
+                        congenital_flag = int(congenital_flag)
+                    except TypeError:
+                        log.info(f"Error, 'hs_condition_is_congenital' is not populated in {table_name}")
+                    row[entity_name] = '-'.join([row.get('dog_id'), row.get('hs_condition'), congenital_flag])
                     # store data
                     column_set.update(row.keys())
                     row_list.append(row)
                 else:
-                    print(f"Unrecognized table: {table_name}")
+                    log.info(f"Unrecognized table: {table_name}")
 
     # make sure pk is the first column
     # pop out the PK, will be splitting this set out later
     column_set.remove(entity_name)
```

Comment:
i'm worried about this line because it's referring to a variable defined in a lower scope than this, so it's difficult to reason about what value it would have.